src/oogeso/core/devices/separator.py

```diff
@@ -29,7 +29,7 @@ def _rule_separator(self, pyomo_model: pyo.Model, t: int, i: int) -> Union[pyo.E
         dev_data = self.dev_data
         node = dev_data.node_id
         node_obj: NetworkNode = self.optimiser.all_nodes[node]  # Fixme: Unresolved reference
-        wellstream_prop: dto.CarrierWellstreamData = self.all_carriers["wellstream"]  # Fixme: Unresolved reference
+        wellstream_prop: dto.CarrierWellStreamData = self.all_carriers["wellstream"]  # Fixme: Unresolved reference
```

Comment:
We need to resolve self.optimiser.all_nodes and self.all_varriers

Reply:
also fixed in pull request test: Unit tests for pyomo model device constraints #74